COM-656 Add new section for index testing

tmvumbi2 opened the pull request:
Jira ticket: https://jembiprojects.jira.com/browse/COM-656

tmvumbi2 committed:
COM-656 Add new section for index testing
Adds a new section for index testing that is folded by default

lauravignoli commented on ui/app/registration/views/patientcommon.html:
```diff
@@ -278,6 +278,24 @@
+        <div ng-show="patient.hasIndexInfo" ng-repeat="attributeRow in ::patientConfiguration.customAttributeRows()">
+            <article style="margin: 0" class="form-field" ng-repeat="attribute in ::attributeRow"
+                ng-class="::{'right-form-field' : $index%2 != 0}" ng-if="attribute.name == 'PERSON_ATTRIBUTE_TYPE_INDEX_KNOW_VIH_STATUS'">
+                <attribute-types target-model="patient" attribute="::attribute" field-validation="::fieldValidation"
```
@tmvumbi2 Fetching attributes like this only shows the ones that are not hidden. And if they are not hidden, they will show in the Other Information section. Please check the death information section to investigate ways of fetching the attributes in some other way.

lauravignoli commented on ui/app/registration/views/patientcommon.html:
```diff
@@ -278,6 +278,24 @@
+              <strong><a ng-bind-html="::'REGISTRATION_TITLE_INDEX_TESTING_CONTACT_INFO' | translate"></a></strong>
+          </span>
+        </legend>
+
```
@tmvumbi2 Please add some space between the title and the question, and move the indent to the right (as in other information).

![index](https://user-images.githubusercontent.com/1198827/74811722-c514d580-52fa-11ea-85ab-84903d70bdf6.png)

tmvumbi2 committed:
COM-656 Update code to display the person attribute PERSON_ATTRIBUTE_TYPE_INDEX_KNOW_VIH_STATUS
The attribute is now hidden and will not be available in patientConfiguration.customAttributeRows(). This commit uses patientConfiguration.attributeTypes to access the person attribute PERSON_ATTRIBUTE_TYPE_INDEX_KNOW_VIH_STATUS.
